tests/typ/compiler/repr.typ

```diff
@@ -47,3 +47,15 @@
 #int \
 #type("hi") \
 #type((a: 1))
+
+---
+// Repr functions
+
+// just for readability
+#set page(width: 135pt)
```

Comment:
would be nice if this would use typc raw blocks.

also, looking at the output, `f(..(x, y))` is a bit redundant. could we flatten it to `f(x, y)` if we detect that it is an array?

Reply:
smart! and sure absolutly